app/controllers/publish/courses/visa_sponsorship_controller.rb

```diff
@@ -0,0 +1,60 @@
+module Publish
+  module Courses
+    class VisaSponsorshipController < PublishController
+      include CourseBasicDetailConcern
+
+      def new
+        raise NotImplementedError
```

Comment:
Are you using this as a 'not implemented yet' flag?

Reply:
Yes. It should just raise the error if the method isn't implemented